2.overview_and_terminology.md

```diff
@@ -73,8 +73,8 @@ Application scopes provide different ways to group components into applications.
 An application scope is described as:
 
 - Metadata: Information about the scope
-- Type: The type of scope. Different types of scopes provide different capabilities.
-- Parameters: Configuration that may be specified in an application configuration
+- Allow overlap: Specifies whether a component can exist in multiple type of this scope
```

Comment:
Maybe we need to keep the Type of scope? 

Because in OAM impl, `allowComponentOverlap` should only check if given Component is included by Scopes of same type only.

Reply:
What does `type` mean here? Can you give a type example?

Reply:
type: networking

type: healthgroup

type: securitygroup

Reply:
After think it a while I feel in most cases Kind is enough, let's keep it as is.